Changed datafeed URL

alvin-leong opened the pull request:
The datafeed URL in the original has been changed. The old URL is not working, I have changed it to the new one.

Commit:
Changed datafeed URL

zhar97 commented:
<img width="380" alt="image" src="https://github.com/user-attachments/assets/338cb6c6-b285-4202-8ea2-c27fa5bee726" />

validated and tested

However, this is actually a duplicate of #31

Commit:
Throttle Dukascopy hourly fetches

Limit per-day hourly request concurrency and add retry backoff so Colab runs are less likely to trigger transient Dukascopy 503 responses.

Made-with: Cursor

Commit:
Harden Dukascopy retries for Colab

Use more conservative defaults and configurable timeout settings to reduce read timeouts from shared Colab networks.

Made-with: Cursor
